S32K/canfd.cpp

```diff
@@ -0,0 +1,836 @@
+/*
+ * Copyright (c) 2020, NXP. All rights reserved.
+ * Distributed under The MIT License.
+ * Author: Abraham Rodriguez <abraham.rodriguez@nxp.com>
+ */
+
+/** @file
+ * Source driver file for the media layer of Libuavcan v1 targeting
+ * the NXP S32K14 family of automotive grade MCU's running
+ * CAN-FD at 4Mbit/s data phase and 1Mbit/s in nominal phase.
+ */
+
+#include "canfd.hpp"
```

Comment:
```suggestion
#include "libuavcan/media/S32K/canfd.hpp"
```

Reply:
Added in https://github.com/noxuz/platform_specific_components/commit/f86edb519ec3bd86aff1aade093e4e3084f4c009